device_backends/include/DeviceBackend.h

```diff
@@ -96,6 +99,12 @@ namespace ChimeraTK {
      * the appropriate ChimeraTK::runtime_error is thrown by this function.
      */
     virtual void checkActiveException() = 0;
+
+    /**
+     * Create an InterruptControllerHandler for a specific controllerID and a TriggerDistributor as parent.
+     */
+    virtual boost::shared_ptr<InterruptControllerHandler> createInterruptControllerHandler(
```

Comment:
Will be renamed later when InterruptControllerHandler is renamed to MuxedInterruptDistributor.